client/src/components/Messaging/Conversation.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import {
+    Card,
+    CardActionArea,
+    CardActions,
+    CardContent,
+    CardMedia,
+    Button,
+    Typography,
+    makeStyles
+} from "@material-ui/core";
+import {Link} from 'react-router-dom';
+
+const useStyles = makeStyles((theme) => ({
+
+    media: {
+        display: 'flex',
+        width: '100vw',
+    },
+    root: {
+        display: 'inline',
+        padding: theme.spacing(1),
+        maxWidth: '100%',
+        marginLeft: "20px",
+        marginBottom: '20px',
+        backgroundColor: "#d8e3e6",
+        minWidth: '300px',
+    },
+    seller: {
+        display: 'inline',
+        textAlign: 'center',
+        width: '50vw',
+        backgroundColor: '#c2debd',
+        color: '#3d804f',
+    },
+    details: {
+        display: 'inline',
+        textAlign: 'center',
+        width: '50vw',
+        backgroundColor: '#c2debd',
+        color: '#3d804f',
+    },
+}));
+
+export default function PopularItem({ event }) {
```

Comment:
Why is the component for a conversation named PopularItem?

Reply:
These commits were not supposed to be on this pull request. It was supposed to be for the accessibility additions only. I was copying our code from loading product details to attempt to get messages to store and display. I changed the naming, but it is still incomplete. I will try to change the pull request.